src/main/migration/20220404222042-add-orderNum-column.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addColumn("WorkoutExercise", "orderNum", {
+      type: Sequelize.INTEGER.UNSIGNED,
+      allowNull: false
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    /**
+     * Add reverting commands here.
```

Comment:
Could you add the down vers with `removeColumn` just in case we need it in the future? And also specify default value to be 0 or allow for null- cause we probably don't want to drop all our tables each time!